<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VetFormMetricsDialog</class>
 <widget class="QDialog" name="VetFormMetricsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>243</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form Metrics For Vet Manager</string>
  </property>
  <widget class="Q3Table" name="metricsTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>380</width>
     <height>140</height>
    </rect>
   </property>
   <property name="numRows">
    <number>5</number>
   </property>
   <property name="numCols">
    <number>2</number>
   </property>
   <row>
    <property name="text">
     <string>SD Certificate</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>MN Certificate</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Bruc. Certificate</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>TB Certificate</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>ALLFLEX Report</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Start</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Height</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">dao/setupdao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
